### PR TITLE
Helm chart warning

### PR DESCRIPTION
**Pull request by rsalmond**
**Please provide a description of this PR:**

It seems to be somewhat common for folks to try using the helm charts from master instead of the helm repo. This might dissuade them.

### CODE REVIEW

File: manifests/charts/README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Istio Installer
 
+## WARNING: Do not use the files in this directory to install Istio!
+
+This directory contains the helm chart _sources_ which are versioned and built and pushed to the helm
+repository with each Istio release. If you want to make changes to Istio helm charts, you're in the
+right place. If you are just looking to install Istio using helm please use the charts found in the
+Istio helm repository documented [here](https://istio.io/latest/docs/setup/install/helm/). If you are
+seeing image pull errors about being unable to pull a container called `image: auto` this is why.
+
```
Comment (bleggett) [inline manifests/charts/README.md:10]:
```suggestion
## WARNING: Do not use the files in this directory to install Istio!

This directory contains the helm chart _sources_ which are versioned, built and pushed to following helm
repositories with each Istio release.  If you want to make changes to Istio helm charts, you're in the
right place. 

If you want to install Istio with Helm, instead please [follow the Helm installation docs here](https://istio.io/latest/docs/setup/install/helm/). If you are seeing image pull errors about being unable to pull a container called `image: auto`, this is why.
 
Charts in this folder are published to the following Helm repos:
 - `https://istio-release.storage.googleapis.com/charts` (charts for official release versions) 
 - `oci://gcr.io/istio-release/charts/` (charts for official release versions and dev build versions)
 
Chart publishing is handled by https://github.com/istio/release-builder
```

**Review by bleggett — COMMENTED** [review]
It's a good idea, added some more context we've been missing in docs as well.

File: manifests/charts/README.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Istio Installer
 
+## WARNING: Do not use the files in this directory to install Istio!
+
+This directory contains the helm chart _sources_ which are versioned and built and pushed to the helm
+repository with each Istio release. If you want to make changes to Istio helm charts, you're in the
+right place. If you are just looking to install Istio using helm please use the charts found in the
+Istio helm repository documented [here](https://istio.io/latest/docs/setup/install/helm/). If you are
+seeing image pull errors about being unable to pull a container called `image: auto` this is why.
```
Comment (howardjohn) [inline manifests/charts/README.md:9]: That is not what `auto` comes from. That means no injection, and can happen with prod images/charts. see https://artifacthub.io/packages/helm/istio-official/gateway#image-auto-information

**Review by howardjohn — COMMENTED** [review]
LGTM sans my one comment

**Review by howardjohn — APPROVED** [review]

**Pull request merged**